tests/unit/RestorationBusinessInformation.spec.ts

```diff
@@ -1,18 +1,49 @@
 import { shallowWrapperFactory } from '../jest-wrapper-factory'
 import { RestorationBusinessInformation } from '@/views'
 import { RestorationResources } from '@/resources/'
+import OfficeAddresses from '@/components/common/OfficeAddresses.vue'
+import BusinessContactInfo from '@/components/common/BusinessContactInfo.vue'
 
-describe('Restoration Review Confirm view', () => {
-  it('renders the component properly', () => {
-    const wrapper = shallowWrapperFactory(
-      RestorationBusinessInformation,
-      null,
-      null,
-      null,
-      RestorationResources
-    )
+// Test Case Data
+const restorationBusinessInfo = [
+  {
+    entityType: 'BEN'
+  },
+  {
+    entityType: 'BC'
+  },
+  {
+    entityType: 'ULC'
+  },
+  {
+    entityType: 'CC'
+  }
+]
```

Comment:
Added all types of entities in case functionality changes in later

Reply:
Nice! 👍